src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -615,7 +615,13 @@ public Response getEventBookingCSV(@Context final HttpServletRequest request,
             for (EventBookingDTO booking : eventBookings) {
                 ArrayList<String> resultRow = Lists.newArrayList();
                 UserSummaryDTO resultUser = booking.getUserBooked();
-                RegisteredUserDTO resultRegisteredUser = this.userAccountManager.getUserDTOById(resultUser.getId());
+                RegisteredUserDTO resultRegisteredUser;
+                try {
+                    resultRegisteredUser = this.userAccountManager.getUserDTOById(resultUser.getId());
+                } catch (NoUserException e) {
+                    log.error(String.format("User with ID \"%d\" could not be retrieved. Continuing...", resultUser.getId()));
```

Comment:
This isn't helpful, since we can commonly expect this case and it is not an error. Either replace it with a `log.debug(...)` or with a comment to say "// Deleted user, skipping`?